README.md

```diff
@@ -1 +1,63 @@
-# module-ballerina-system
\ No newline at end of file
+Ballerina System Library
+===================
+
+  [![Build](https://github.com/ballerina-platform/module-ballerina-system/workflows/Build%20master%20branch/badge.svg)](https://github.com/ballerina-platform/module-ballerina-system/actions?query=workflow%3ABuild)
+  [![Daily build](https://github.com/ballerina-platform/module-ballerina-system/workflows/Daily%20build/badge.svg)](https://github.com/ballerina-platform/module-ballerina-system/actions?query=workflow%3A%22Daily+build%22)
+  [![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerina-system.svg)](https://github.com/ballerina-platform/module-ballerina-system/commits/master)
+  [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+The System library is one of the standard library modules of the<a target="_blank" href="https://ballerina.io/"> Ballerina</a> language.
+
+This module provides functions to retrieve information about the system and the current users of the system.
+
+For more information go to [The System Module](https://ballerina.io/swan-lake/learn/api-docs/ballerina/system/).
+
+## Building from the Source
+
+### Setting Up the Prerequisites
+
+1. Download and install Java SE Development Kit (JDK) version 8 (from one of the following locations).
+   * [Oracle](https://www.oracle.com/java/technologies/javase/javase-jdk8-downloads.html)
+   
+   * [OpenJDK](http://openjdk.java.net/install/index.html)
+   
+        > **Note:** Set the JAVA_HOME environment variable to the path name of the directory into which you installed JDK.
+     
+### Building the Source
+
+Execute the commands below to build from source.
+
+1. To build the library:
+
+        ./gradlew clean build
+
+2. To run the integration tests:
+
+        ./gradlew clean test
```

Comment:
Did you test this command? I think we need to add `test` task to build gradle to make this work

Reply:
Added the task to the gradle